app/routes/index.js

```diff
@@ -0,0 +1,9 @@
+import Ember from 'ember';
+
+const { Route } = Ember;
+
+export default Route.extend({
+  beforeModel() {
+    this.transitionTo('rounds.index');
```

Comment:
Question: what's the main difference between the index route and the application route?